src/enrichers/privacy-config.js

```diff
@@ -0,0 +1,10 @@
+import { isNonEmpty } from '../utils/types'
+
+export function enrich (state) {
+  if (isNonEmpty(state) && isNonEmpty(state.gdprApplies)) {
+    const privacyMode = !!state.gdprApplies
+    return {
+      privacyMode: privacyMode
```

Comment:
It could sense to make privacyMode an enum so it's easier to expand for ccpa support later

Reply:
We could do it. Though for now it is a boolean indicating that we run in privacy mode independently of the actual privacy framework.